src/components/Cards.js

```diff
@@ -0,0 +1,52 @@
+import imgNone from "../imgs/이미지 없을 때 배경.svg";
+
+function dateDiff(createdAt) {
+  const present = Date.now();
+  const createdLinkDate = new Date(createdAt);
+  const diff = present - createdLinkDate;
+  if (diff < 3600000) {
+    return `${Math.floor(diff / 1000 / 60)} minutes ago`;
+  } else if (diff < 86400000) {
+    return `${Math.floor(diff / 1000 / 60 / 60)} hours ago`;
+  } else if (diff < 2628000000) {
+    return `${Math.floor(diff / 1000 / 60 / 60 / 24)} days ago`;
+  } else if (diff < 31540000000) {
+    return `${Math.floor(diff / 1000 / 60 / 60 / 24 / 30)} months ago`;
+  } else {
+    return `${Math.floor(diff / 1000 / 60 / 60 / 24 / 30 / 12)} years ago`;
+  }
```

Comment:
r: 가독성을 해치는 값들은 상수화해주세요.

```suggestion
  // 예시
  const MINUTE_TO_MS = 3600000;
  const HOUR_TO_MS = 86400000;
  const DAY_TO_MS = 2628000000;
  const MONTH_TO_MS = 31540000000;
  
  if (diff < MINUTE_TO_MS) {
    return `${Math.floor(diff / 1000 / 60)} minutes ago`;
  } else if (diff < HOUR_TO_MS) {
    return `${Math.floor(diff / 1000 / 60 / 60)} hours ago`;
  } else if (diff < DAY_TO_MS) {
    return `${Math.floor(diff / 1000 / 60 / 60 / 24)} days ago`;
  } else if (diff < MONTH_TO_MS) {
    return `${Math.floor(diff / 1000 / 60 / 60 / 24 / 30)} months ago`;
  } else {
    return `${Math.floor(diff / 1000 / 60 / 60 / 24 / 30 / 12)} years ago`;
  }
```